canonical/jaas-dashboard
# WD-1053 - Create general slice

## Done

- Move general actions, reducers, types and selectors to a slice.
- Add tests for the above.

## QA

- Pull code
- Run `dotrun clean && dotrun serve`
- Open http://0.0.0.0:8036/
- Run the app and check that you can log in and list models etc.

## Details

https://warthogs.atlassian.net/browse/WD-1053
